src/screens/hardware-wallets/PairHardwareWalletAgainSheet.tsx

```diff
@@ -45,96 +68,125 @@ export const PairHardwareWalletAgainSheet = () => {
   }));
 
   return (
-    <Layout>
-      <Box style={{ zIndex: 1 }}>
-        <Inset horizontal="36px">
-          <Stack alignHorizontal="center" space="20px">
-            <Text align="center" color="label" weight="bold" size="26pt">
-              {i18n.t(TRANSLATIONS.looking_for_devices)}
-            </Text>
-            <Stack space="10px">
-              <Text
-                align="center"
-                color="labelTertiary"
-                weight="semibold"
-                size="15pt / 135%"
-              >
-                {i18n.t(TRANSLATIONS.make_sure_bluetooth_enabled)}
+    <>
+      <Layout>
+        <Box style={{ zIndex: 1 }}>
+          <Inset horizontal="36px">
+            <Stack alignHorizontal="center" space="20px">
+              <Text align="center" color="label" weight="bold" size="26pt">
+                {hardwareTXError
+                  ? i18n.t(TRANSLATIONS.transaction_rejected)
+                  : isReady
+                  ? i18n.t(TRANSLATIONS.confirm_on_device)
+                  : i18n.t(TRANSLATIONS.looking_for_devices)}
```

Comment:
Not a big fan of this nested ternary, could it be prepared differently? Like stored in a constant or just having a helper like `getPairHardwareWalletAgainFooCopy`?